phpspy.c

```diff
@@ -199,15 +230,15 @@ static void parse_opts(int argc, char **argv) {
     ) {
         switch (c) {
             case 'h': usage(stdout, 0); break;
-            case 'p': opt_pid = atoi(optarg); break;
+            case 'p': opt_pid = atoi_with_min_or_exit("-p", optarg, 1); break;
             case 'P': opt_pgrep_args = optarg; break;
-            case 'T': opt_num_workers = atoi(optarg); break;
-            case 's': opt_sleep_ns = strtol(optarg, NULL, 10); break;
-            case 'H': opt_sleep_ns = (1000000000UL / strtol(optarg, NULL, 10)); break;
+            case 'T': opt_num_workers = atoi_with_min_or_exit("-T", optarg, 1); break;
+            case 's': opt_sleep_ns = strtol_with_min_or_exit("-s", optarg, 1); break;
+            case 'H': opt_sleep_ns = strtol_with_min_or_exit("-H", optarg, 1); break;
```

Comment:
Should be `(1000000000UL / strtol_with_min_or_exit("-H", optarg, 1))`

Reply:
This should be fixed